src/application-base/ApplicationBase.jsx

```diff
@@ -134,9 +135,11 @@ const ApplicationBase = ({
                     }}
                   >
                     <ApplicationLoadingOverlayProvider>
-                      <Suspense fallback={<ApplicationLoadingOverlay isOpen />}>
-                        {children}
-                      </Suspense>
+                      <BannerCheckpoint fitToParentIsDisabled={fitToParentIsDisabled}>
```

Comment:
We need to discuss where we want these banners in the application hierarchy. I don't think it hurts to have them here as a backup but primarily i'd imagine these show up in page layout content and in modals.

Reply:
I agree. I'm not sue if/when banners would be at this level when navigation & page layouts are provided in the application. We could potentially remove it from this level.